index.html

```diff
@@ -143,106 +120,6 @@ <h2>Contact me</h2>
-               <div class="lang-btn">
-                  <ul>
-                     <li><a href="#"><button>html</button></a></li>
-                     <li><a href="#"><button>css</button></a></li>
-                     <li><a href="#"><button>javaScript</button></a></li>
-                  </ul>
-               </div>
-               <div class="see-project-btn"><button>See Project</button></div>
-            </div>
-         </section>
-         <section class="desktop-card">
-            <div class="desk_main_content">
-               <h2>Multi-Post Stories</h2>
-               <div class="canopy"></div>
-               <p>Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.</p>
-               <div class="lang-btn">
-                  <ul>
-                     <li><a href="#"><button>html</button></a></li>
-                     <li><a href="#"><button>css</button></a></li>
-                     <li><a href="#"><button>javaScript</button></a></li>
-                  </ul>
-               </div>
-               <div class="see-project-btn"><button>See Project</button></div>
-            </div>
-            <div class="desk_main_img2"></div>
-         </section>
-         <section class="desktop-card">
-            <div class="desk_main_img3"></div>
-            <div class="desk_main_content">
-               <h2>Multi-Post Stories</h2>
-               <div class="canopy"></div>
-               <p>Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.</p>
-               <div class="lang-btn">
-                  <ul>
-                     <li><a href="#"><button>html</button></a></li>
-                     <li><a href="#"><button>css</button></a></li>
-                     <li><a href="#"><button>javaScript</button></a></li>
-                  </ul>
-               </div>
-               <div class="see-project-btn"><button>See Project</button></div>
-            </div>
-         </section>
-         <section class="desktop-card">
-            <div class="desk_main_content">
-               <h2>Multi-Post Stories</h2>
-               <div class="canopy"></div>
-               <p>A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.</p>
-               <div class="lang-btn">
-                  <ul>
-                     <li><a href="#"><button>html</button></a></li>
-                     <li><a href="#"><button>css</button></a></li>
-                     <li><a href="#"><button>javaScript</button></a></li>
-                  </ul>
-               </div>
-               <div class="see-project-btn"><button>See Project</button></div>
-            </div>
-            <div class="desk_main_img4"></div>
-         </section>
-      </section>
```

Comment:
- [ ] `required`
- [ ] Note ❗ this also applies to the mobile pop-up cards
- [ ] Good job with the pop-up cards 👏 
- [ ] However there is just a few things you need to address
- [ ] When I click on the see project button the pop-up is opening all the cards at the same time instead of only opening only the card i wanted to open
- [ ] also please add some blur in the background like the figma template
- [ ] And lastly i suggest you also make the pop fit the screen where i dont have to scroll up to look for the close button just as the figma template 
- [ ] Please observe the screenshots and gif

`Your pop-up card gif` `click on the gif to open it`
![Animation2](https://user-images.githubusercontent.com/66230306/232233066-1d384418-d162-44c6-bc9f-23a920dca6ae.gif)

`figma template`
![image](https://user-images.githubusercontent.com/66230306/232233468-8b5bd8c9-49c8-4c5f-a4ff-405a2c149f1a.png)